scripts/build-css

```diff
@@ -2,9 +2,8 @@
 
 set -e # exit when error
 
-VERSION=$(json version < package.json)
-NAME='docsearch.js'
-LICENSE="/*! ${NAME} ${VERSION:-UNRELEASED} | © Algolia | github.com/algolia/docsearch.js */"
+NAME='docsearch'
```

Comment:
It will work, the version will be "UNRELEASED" in the licence file that's it.

When would you need to do so? (Or just curious?)